test/base/src/jvmTest/kotlin/io/realm/kotlin/test/compiler/EmbeddedTests.kt

```diff
@@ -62,6 +62,6 @@ class EmbeddedTests {
             )
         )
         assertEquals(KotlinCompilation.ExitCode.COMPILATION_ERROR, result.exitCode)
-        assertTrue(result.messages.contains("RealmLists does not support nullable realm objects element type"))
+        assertTrue(result.messages.contains("LISTs do not support nullable realm objects element types"))
```

Comment:
See my comment about error messages further up. This would prevent these changes here.